scm/driver/github/git.go

```diff
@@ -37,6 +37,15 @@ func (s *gitService) FindTag(ctx context.Context, repo, name string) (*scm.Refer
 	return convertRef(out), res, err
 }
 
+func (s *gitService) CreateBranch(ctx context.Context, repo string, params *scm.CreateBranch) (*scm.Response, error) {
+	path := fmt.Sprintf("repos/%s/git/refs", repo)
+	in := &createBranch{
+		Ref: fmt.Sprintf("refs/heads/%s", params.Name),
```

Comment:
we have a helper function that might be useful here:

```
scm.ExpandRef("refs/heads", params.Name)
```

Reply:
change made, thanks for the reminder. 👍